src/Framework/Framework/Binding/ExtractGenericArgumentDataContextChangeAttribute.cs

```diff
@@ -48,11 +49,11 @@ public ExtractGenericArgumentDataContextChangeAttribute(Type genericType, int ty
         var implementations = ReflectionUtils.GetBaseTypesAndInterfaces(dataContext).ToList();
         if (implementations.Count == 0)
         {
-            throw new Exception($"The data context {dataContext} doesn't implement {GenericType}!");
+            throw new Exception($"The data context {dataContext} doesn't implement {GenericType.CSharpFullName}!");
```

Comment:
```suggestion
            throw new Exception($"The data context {dataContext.ToCode()} doesn't implement {GenericType.CSharpFullName}!");
```

Not your mistake, just noticed :]